src/orocos_cpp.cpp

```diff
@@ -129,9 +130,9 @@ bool OrocosCpp::initialize(const OrocosCppConfig& config, bool quiet)
             return false;
         }
         if(config.create_log_folder){
-            st = set_env("ORO_LOGFILE", bundle->getLogDirectory()+"/orocos.log", true);
+            set_env("ORO_LOGFILE", bundle->getLogDirectory()+"/orocos-"+boost::lexical_cast<std::string>(getpid())+".log", true);
```

Comment:
any reason for using different name than here https://github.com/rock-cpp/orocos_cpp/blob/8544ac715d3b6594d771ff82cdc25f24de449912/src/Spawner.cpp#L147

Reply:
@maltewi is this ok to merge like it is?